backend.native/tests/build.gradle

```diff
@@ -3318,6 +3318,14 @@ if (PlatformInfo.isAppleTarget(project)) {
         it.headers "$projectDir/interop/objc/msg_send/messaging.h"
         it.linkerOpts "-L$buildDir", "-lobjcmessaging"
     }
+
+    createInterop("objcKt34467") {
+        it.defFile 'interop/objc/kt34467/module_library.def'
+        def moduleMap = file("interop/objc/kt34467/module_library.modulemap").absolutePath
+        it.compilerOpts "-fmodule-map-file=$moduleMap"
+        // We need to import CoreFoundation.h but there is no need to run interop for it.
```

Comment:
Can we make the test import our own header?
Making the test dependent on platform libs is somewhat fragile.

Reply:
Fixed.